topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/rest/HealthActuatorContractTest.java

```diff
@@ -30,31 +30,31 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * An external requirement of the service is to keep-alive in k8s.
- * This is done by checking the service is still alive and healthy by REST GET /actuator/health.
- * This should return 200 OK if the service is healthy.
+ * SpringBoot Starter Actuator is a service loaded-up by adding the spring-boot-starter-actuator as a dependency to the
+ * project and configured in the application.yaml file.  It is a service which provides information on the application
+ * and is being used to monitor the "health" of the palisade service.  If there is an indication that this service has
+ * fallen over, this information can be used to restore the service.
```

Comment:
We dont need to explain what the SpringBoot starter actuator is, saying that this test checks the services health endpoint is enough

Reply:
Ok